metron-docker/kafkazk/bin/init-zk.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+echo "create /metron metron" | ./bin/zookeeper-shell.sh localhost:2181
+echo "create /metron/topology topology" | ./bin/zookeeper-shell.sh localhost:2181
+echo 'create /metron/topology/global {"solr.zookeeper":"metron-kafkazk:2181","solr.collection":"metron","solr.numShards":1,"solr.replicationFactor":1,"es.clustername":"elasticsearch","es.ip":"metron-elasticsearch","es.port":"9300","es.date.format":"yyyy.MM.dd.HH"}' | ./bin/zookeeper-shell.sh localhost:2181
```

Comment:
I get the creations of the directories in zk, but why are we pushing configs here instead of using the zk_load_utils shell script to submit the ones we deploy?